pkg/cmd/server/start/start_node.go

```diff
@@ -412,7 +412,7 @@ func execKubelet(kubeletArgs []string) error {
 	for i, s := range os.Args {
 		if s == "--vmodule" {
 			if i+1 < len(os.Args) {
-				args = append(args, fmt.Sprintf("--vmodule=", os.Args[i+1]))
+				args = append(args, fmt.Sprintf("--vmodule=%s", os.Args[i+1]))
```

Comment:
Isn't there the potential that this needs to be quoted?

Reply:
Yup, it may.

Reply:
Actually, from what I've looked into the code, it doesn't have to. We're safe with this approach.